docs/src/ensemble_kalman_inversion.md

```diff
@@ -64,18 +83,19 @@ A call to the inversion algorithm can be performed with the `update_ensemble!` f
 
 A typical use of the `update_ensemble!` function given the ensemble Kalman inversion object `ekiobj`, the dynamical model `Ψ` and the observation map `H` is
 ```julia
-N_iter = 20 # Number of steps of the algorithm
+N_iterations = 20 # Number of steps of the algorithm
```

Comment:
We have a glossary of terms for consistency amongst documentation and code for the inverse problem examples.

https://clima.github.io/EnsembleKalmanProcesses.jl/dev/glossary/

Though it is fine to change `N_iter` to `N_iteration` if you believe it is needed, let's do it in a separate PR, as it needs to be changed everywhere.

Reply:
It's not *needed*, but what is really needed? We can get philosophical here. It's just much easier to read code if it's verbose and here 4-5 characters make the semi-incomprehensible `N_iter` to read almost English. :)